docs/user/pings/custom.md

```diff
@@ -6,6 +6,8 @@ Applications can define metrics that are sent in custom pings. Unlike the built-
 
 Custom pings must be defined in a `pings.yaml` file, which is in the same directory alongside your app's `metrics.yaml` file.
 
+Ping names are limited to lowercase ASCII characters and hyphens and a maximum of 30 characters.
```

Comment:
Just a wording nit: 
```suggestion
Ping names are limited to lowercase letters and hyphens with a maximum of 30 characters.
```

Reply:
Why remove "ASCII"?  It's probably important to state that you can't use é, for example.

Reply:
ASCII `0X8A` == `é`

Reply:
Again, just a nit.  I'm fine if `ASCII` stays there, I know what is meant by it and most everyone else would too.

Reply:
Side stepped the whole issue by using the [same wording as in other places](https://docs.telemetry.mozilla.org/cookbooks/new_ping.html).